[W-10823160] Update exchange API docs with icon upload section (#184)
mulesoft/docs-exchange: modules/ROOT/pages/exchange-api.adoc

To use the Exchange API version 1 to create an asset and set its lifecycle state, send the status property as a field.

[[upload-asset-icons]]

> === Upload Asset Icons

Reviewer: ```suggestion
=== Uploading Images to be Used as Asset Icons
```
Suggesting this to avoid the imperative voice in the title, which typically indicates that the content is a task, with steps. Also to be more precise: you upload the images, which are then used as icons.

Reviewer: Do we need to say anything about why the audience wants or needs to do this: advantage?

Reviewer 2: I can add that these icons display in Exchange which may be implied though.

> You can upload an image to use as an asset icon through the publish endpoint. The following file types are supported:

Reviewer: Alternative to omit modal verb "can":
```suggestion
Upload an image to use as an asset icon by using the publish endpoint. The following file types are supported:
```

Reviewer 2: I committed Kim's suggestion.

* GIF
* JPG
* JPEG
* PNG
* SVG

> Asset icons are version agnostic. If an icon exists for a previous version of an asset, you cannot use another icon for a subsequent asset version. The following example shows how to upload an asset icon. To specify the asset, replace the identifiers `:groupId`, `:assetId`, and `:version`. Ensure that the package icon format `files.icon.png` has the same file type as the icon file `my-asset-icon.png`. PNG is the file type in this example.

Reviewer: ```suggestion
Asset icons are version agnostic. They are not replaced by new icons when assets are versioned. 
+
The following example shows how to upload a .png file image to use as an asset icon. To specify the asset, replace the identifiers `:groupId`, `:assetId`, and `:version` with values specific to your implementation. Ensure that `files.icon.png` has the same file type as `my-asset-icon.png`.
```

Reviewer 2: Committed Kim's suggestion.

To use the Exchange API version 1 to create an asset and set its lifecycle state, send the status property as a field.

[[upload-asset-icons]]
=== Upload Asset Icons

You can upload an image to use as an asset icon through the publish endpoint. The following file types are supported:

* GIF
* JPG
* JPEG

> * PGN

Author: typo: PNG

* SVG

> Asset icons are version agnostic. If an icon exists for a previous version of an asset, uploading a new icon for a subsequent asset version is ignored. The following example shows how to upload an asset icon. To specify the asset, replace the identifiers `:groupId`, `:assetId`, and `:version`. Ensure that the package format `files.icon.png` has the same file type as the icon file `my-asset-icon.png`.

Author: Maybe I'd clarify which part is the format:

Ensure that the package format (`png` in `files.icon.png`) has the same file type as the icon file (`my-asset-icon.png`).
